src/mqttgw.erl

```diff
@@ -45,6 +45,8 @@
     on_client_gone/1
 ]).
 
+-export([send_dynsub_response/7]).
```

Comment:
Can we also move this function to `mqttgw_dyn_srv`? We can reuse it from there in the plugin main module.